### PR TITLE
chore: add notice to readme

### PR DESCRIPTION
**Pull request by superbogy**

### CODE REVIEW

**Comment by superbogy** [comment]
#112

File: README.md [hunk 1]
```diff
@@ -7,12 +7,39 @@ This is a Couchbase connector node module for [Loopback](http://loopback.io/) wi
...
+
+ ```
+ update = Object.assign(current, incoming)
+ ```
+
+Discussion of `update` and `updateAttribetes` , see: [https://groups.google.com/forum/#!topic/loopbackjs/-1jarvOuh8k](https://groups.google.com/forum/#!topic/loopbackjs/-1jarvOuh8k)
+
+- since version 1.0.0 the default vaule of  `stale` was set to `ok` in viewQuery.
+
+If the update condition field isn't `id`  you must put `stale` in `options` parameter to make sure you get the correct data, like:
```
Comment (xavierchow) [inline README.md]:
```suggestion
If the update condition field isn't `id`  you might need to set `stale` in `options` parameter to make sure you get the latest data if you have a strong need of data instant consistency, like:
```

File: README.md [hunk 2]
```diff
@@ -7,12 +7,39 @@ This is a Couchbase connector node module for [Loopback](http://loopback.io/) wi
...
 ## How to use
 
+
 ### Install
 
 ```
 npm install loopback-connector-couchbasex --save
 ```
 
+### Notice
```
Comment (xavierchow) [inline README.md]:
```suggestion
### Caveat
```

File: README.md (hunk 1 above)
Comment (superbogy) [reply README.md]: I think `stale` must be set to 1 in a write operation. Otherwise, the dirty read and dirty write would happen.

Comment (xavierchow) [reply README.md]: k, I got you, you are talking about `write`, LGTM.

**Pull request merged**